src/components/navigation/navigation.tsx

```diff
@@ -6,12 +6,12 @@ import { AiOutlineHeart, AiOutlineSearch, AiOutlineShoppingCart } from "react-ic
 
 export const Navigation = () => {
   return (
-    <nav className="bg-whitea-13 fixed bottom-0 z-50 w-full sm:sticky sm:w-auto sm:self-start">
+    <nav className="fixed bottom-0 z-50 w-full bg-whitea-13 sm:sticky sm:w-auto sm:self-start">
```

Comment:
既存の問題ですが、サイドバーの背景色が真っ白になったままだったので、この際修正してもらえるとありがたいです🙏
（めっちゃわかりづらいですが、サイドバーの周りだけ微妙に色が違います）
```suggestion
    <nav className="fixed bottom-0 z-50 w-full bg-mauve-1 sm:sticky sm:w-auto sm:self-start">
```

Reply:
@takeyu1013 
修正しました！